### PR TITLE
Powerup/boost

### PR DESCRIPTION
**Pull request by mikeslgoh**
Last feature to add from build day:
- Added Boost to object pooler and equip with key 5
- Made sure every powerup has an icon
- Renamed the project to PartyParkade
- Also made it that the image and text will be hidden when there is no powerup equipped

### CODE REVIEW

File: Assets/Script/ScriptableObject/Boost.asset [hunk 1]
```diff
@@ -10,8 +10,8 @@ MonoBehaviour:
   m_Enabled: 1
   m_EditorHideFlags: 0
   m_Script: {fileID: 11500000, guid: 9f37146a2dab840debabee1ee0fb9738, type: 3}
-  m_Name: BouncyObjectPowerUp
+  m_Name: Boost
```
Comment (m4rtint) [inline Assets/Script/ScriptableObject/Boost.asset]: How come this asset is renamed?

Comment (m4rtint) [reply Assets/Script/ScriptableObject/Boost.asset]: Oh nvm

File: Assets/Script/UI/PlayerUserInterface.cs [hunk 2]
```diff
@@ -70,6 +70,15 @@ public void SetPowerUpUI (PowerUpProperties properties)
         {
             powerName = properties.Name;
             powerUpSprite = properties.Icon;
+            if ((int)m_IconOfPowerUp.color.a == 0)
+            {
+                m_IconOfPowerUp.color = new Color(1,1,1,1);
```
Comment (m4rtint) [inline Assets/Script/UI/PlayerUserInterface.cs]: Is it not possible to make the sprite/image null? 
instead of making it alpha?

Comment (mikeslgoh) [reply Assets/Script/UI/PlayerUserInterface.cs]: It’s not really the image being empty I believe. It seems that the image default is a white background which is why I tried changing it to be transparent.

File: Assets/Script/UI/PlayerUserInterface.cs [hunk 3]
```diff
@@ -66,14 +66,17 @@ public void SetPowerUpUI (PowerUpProperties properties)
     {
         string powerName = string.Empty;
         Sprite powerUpSprite = null;
+        Color spriteColor = Color.clear;
```
Comment (mikeslgoh) [inline Assets/Script/UI/PlayerUserInterface.cs]: nice addition @m4rtint

**Pull request merged**